kanin/src/request.rs

```diff
@@ -17,9 +17,9 @@ pub struct Request<S> {
     channel: Channel,
     /// Request ID. This is a unique ID for every request. Either a newly created UUID or whatever
     /// is found in the `req_id` header of the incoming AMQP message.
-    pub(crate) req_id: ReqId,
```

Comment:
Any particular reason for this? Or just personal pref?

Reply:
Mostly just personal preference and for "fairness". kanin should implement extract in the same way as a downstream crate.